.pre-commit-config.yaml

```diff
@@ -65,3 +65,9 @@ repos:
     rev: v5.9.3
     hooks:
     -   id: isort
+
+-   repo: https://github.com/asottile/blacken-docs
```

Comment:
Why that repo? I recently found
```
-   repo: https://github.com/ambv/black
    rev: stable
    hooks:
    -   id: black
        language_version: python3.7
```

Reply:
changed it to https://github.com/ambv/black